src/components/SideBar/Brands/Brands.jsx

```diff
@@ -0,0 +1,21 @@
+import { useContext } from "react";
+import Heading from "../Heading/Heading";
+import './styles.scss';
+import { ProductsContext } from "../../../ProductsContext";
+import CheckBox from "../CheckBox/CheckBox";
+
+const Brands = () => {
+  const productContext = useContext(ProductsContext);
+
+  const productBrands = new Set(productContext.products.map(product => product.brand));
```

Comment:
```suggestion
  const productBrands = [...new Set(productContext.products.map(product => product.brand))];
```
Như này là productBrands nó sẽ thành ra 1 mảng rồi